src/lib.rs

```diff
@@ -419,6 +419,39 @@ pub fn get_all_enclave_names() -> NitroCliResult<Vec<String>> {
     Ok(objects)
 }
 
+/// Sends the name to all the running enclaves and expects a response
+/// with the ID of the one that uniquely matched
```

Comment:
What guarantees that names are unique? I haven't seen any check in run-enclave that ensures this.

Reply:
I've searched a bit more and finally found it.